types/staking.go

```diff
@@ -25,17 +25,21 @@ const (
 	// Constant as this should not change without a hard fork.
 	// TODO: Link to some Tendermint docs, this is very unobvious.
 	ValidatorUpdateDelay int64 = 1
+
+	BondStatusUnbonded  = "Unbonded"
+	BondStatusUnbonding = "Unbonding"
+	BondStatusBonded    = "Bonded"
 )
 
-//BondStatusToString for pretty prints of Bond Status
+// BondStatusToString for pretty prints of Bond Status.
 func BondStatusToString(b BondStatus) string {
```

Comment:
We should modify this to be `func (b BondStatus) String() string`

Reply:
Agreed. I'll update.